book/src/concepts/nested_objects.md

```diff
@@ -7,6 +7,10 @@ SPDX-License-Identifier: MIT OR Apache-2.0
 
 # Nested Objects
```

Comment:
wonder if inheritance and nested objects should be split into another chapter under "Use cases" / "User stories" / "Advanced" / "Complex examples" or something :thinking: 

Then keep the "concepts" to be very general basic concepts like bridge, build systems, duality of C++ and Rust QObject, built-in types